public/css/labels.less

```diff
@@ -0,0 +1,28 @@
+.labels {
+  padding-left: 0;
+
+  .title {
+    display: block;
+    border-radius: .25em;
+    padding: .25em .5em;
+    margin: 0 -.5em;
+
+    background: @gray-lighter;
+  }
+
+  &, ul {
+    list-style-type: none;
+    margin-block-start: 0;
+    margin-block-end: 0;
+  }
+
+  ul {
+    padding-left: 1em;
+  }
+
+  .horizontal-key-value > .key {
+    width: 8em ;
```

Comment:
That `width` has no effect. What happens if a label is longer than this width?

Reply:
Why did you remove it? What is the desired state? I'm not ok with how it looks now as the text is off by 1em compared to the details.